examples/flutter_gallery/lib/demo/pesto_demo.dart

```diff
@@ -78,25 +78,28 @@ class _RecipeGridPageState extends State<RecipeGridPage> {
   @override
   Widget build(BuildContext context) {
     final double statusBarHeight = MediaQuery.of(context).padding.top;
-    return new Theme(
-      data: _kTheme.copyWith(platform: Theme.of(context).platform),
-      child: new Scaffold(
-        key: scaffoldKey,
-        floatingActionButton: new FloatingActionButton(
-          child: const Icon(Icons.edit),
-          onPressed: () {
-            scaffoldKey.currentState.showSnackBar(const SnackBar(
-              content: const Text('Not supported.'),
-            ));
-          },
-        ),
-        body: new CustomScrollView(
-          slivers: <Widget>[
-            _buildAppBar(context, statusBarHeight),
-            _buildBody(context, statusBarHeight),
-          ],
+    return new RouteName(
```

Comment:
nit: the real fix here would probably be to give the Pesto logo on top of this page a label "pesto" and mark that one as routeName. Currently the title logo of the page is not focusable -- but that has nothing to do with this PR.